src/styles/_custom.scss

```diff
@@ -324,15 +324,11 @@ a.has-text-black:hover {
 }
 
 // Content that stays fixed at the bottom of the viewport:
-.jf-footer-menu {
-  width: 100%;
-  position: fixed;
-  bottom: 0;
-  z-index: 30;
-}
-
 .jf-cookies-banner {
   border-top: 1px solid $justfix-black;
+  position: fixed;
+  bottom: 0;
+  width: 100%;
```

Comment:
does this also need the z index?

Reply:
lgtm otherwise!

Reply:
it didn't have one originally, but that was one added when the fixed button was created so I'll put that back to be safe.